book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ ] We have not met one requirement in this class. Please look at the statement below:
  
  > - Create the has-many side of Book and Rental (a book has many rentals).
  
  In the manner you've created the `add_rental` method for the Person class, please do the same for the Book for us to satisfy the above requirement.